src/Microsoft.Sbom.Api/Converters/IManifestPathConverter.cs

```diff
@@ -10,7 +10,8 @@ public interface IManifestPathConverter
         /// that the manifest implements.
         /// </summary>
         /// <param name="path">The relative path of the file.</param>
+        /// <param name="prependDotToPath">If true we will prepend a . before the path.</param>
```

Comment:
>prependDotToPath [](http://example.com/codeflow?start=25&length=16)

Why we need to pre-append a dot?

Reply:
This is how SPDX file paths are stored by default, we add a . to the relative path, so we have to do the same internally for it to match.